packages/data-context/src/sources/ProjectDataSource.ts

```diff
@@ -134,7 +135,7 @@ export function getLongestCommonPrefixFromPaths (paths: string[]): string {
   return lcp.slice(0, endIndex).join(path.sep)
 }
 
-export function getPathFromSpecPattern (specPattern: string, testingType: TestingType, fileExtensionToUse?: 'js' | 'ts') {
+export function getPathFromSpecPattern (specPattern: string, testingType: TestingType, fileExtensionToUse?: 'js' | 'ts', name = '') {
```

Comment:
When we have this many positional arguments, some required, some not, what do you think about moving to an object of options?

```js
interface GetSpecOptions {
  specPattern: string
  testingType: TestingType
  fileExtensionToUse?: 'js' | 'ts'
  name?: string
}
```

Edit: Looks like we've basically got this [here](https://github.com/cypress-io/cypress/pull/23298/files#diff-c351c3ec7e69c56e8c57fdf49babbece37086dbb43b9fc07a386f3cd33e508c9R17), should we just extract an interface and share it?

Also is `fileExtensionToUse` even nullable? It doesn't look like it should be.

Reply:
I ended up not using that interface in both places because the types were slightly different